packages/docs/docs/welcome.md

```diff
@@ -2,7 +2,11 @@
 title: What is UniRep?
 ---
 
-# 👏 Welcome
+# Introduction
+
+UniRep is a private, non-repudiable data storage and retrieval system designed as a base layer for building custom and interoperable zk applications. It serves as a memory layer, allowing users to receive data, prove facts about their data, and store results of large proofs with strong privacy guarantees and constant computation complexity. 
+
+UniRep can be utilized for web2 authentication bridging and storing application-specific data such as user reputation, actions, and memberships. This enables seamless interoperability between applications using publicly available state roots for user proofs.
```

Comment:
I copied another paragraph here because we want the next version to be changed as well.
so if we changed the beta-3 document, we should also change the paragraph here.